packages/api/src/fetchAlerts.ts

```diff
@@ -1,8 +1,10 @@
-import { GetAlertsResponse } from './types';
+import { ApiRequest, GetAlertsResponse } from './types';
 
-async function fetchAlerts(): Promise<GetAlertsResponse | null> {
+export async function fetchAlerts({
+  serverUrl,
+}: ApiRequest<unknown>): Promise<GetAlertsResponse | null> {
```

Comment:
its just there to say we do not expect nothing more than severUrl, so the way the ApiRequest generic works is that it will receive whatever it has and then add on serverUrl. Initially i tried with `{}` but typescript prefers object or unknown and i preferred so the typescript lsp does not recommend anything else